src/react-components/home-root.js

```diff
@@ -12,27 +12,31 @@ import { ENVIRONMENT_URLS } from "../assets/environments/environments";
 import styles from "../assets/stylesheets/index.scss";
 
 import HubCreatePanel from "./hub-create-panel.js";
-import InfoDialog from "./info-dialog.js";
+import ReportDialog from "./report-dialog.js";
+import SlackDialog from "./slack-dialog.js";
+import UpdatesDialog from "./updates-dialog.js";
+import DialogContainer from "./dialog-container.js";
 
 addLocaleData([...en]);
 
 class HomeRoot extends Component {
   static propTypes = {
     intl: PropTypes.object,
-    dialogType: PropTypes.symbol,
+    dialog: PropTypes.string,
```

Comment:
Why is this `string` and not `element` or `node`?

Reply:
Good catch, in the present incarnation it should clearly be `node`. I don't think we even need this prop anymore though.